src/components/layer.js

```diff
@@ -73,7 +72,7 @@ module.exports.Component = registerComponent('layer', {
       this.loadingScreen = false;
     }
 
-    if (type === 'monocubemap') {
+    if (this.layer.layout === 'mono') {
```

Comment:
Was this triggering a bug? Trying to understand because the layer type in the component properties should correspond to `this.layer.layout === 'mono'`. They should be equivalent or we're doing something wrong somewhere else.

Reply:
I would occasionally see it get into loadCubeMapImages() from the tick function, and it would wind up there with the type of 'quad', which would have a mono layout.   It might be worth looking at the overall flow to preventing it from getting there, but this change fixed the error being thrown because of that.

Reply:
I see that function shouldn't run for a `quad` layer type. I added a commit with an alternative logic. Could you verify that also fixes the issue? Sorry to add extra work. Not with my Quest at the moment. Thanks so much!